### ISSUE TITLE
🐛 BUG:  Companion widget open summary and comment focus issue

### ISSUE THREAD

**Issue by Kvaibhav01** [dailydotdev/daily#578]
### What went wrong? 🤔

1. **The widget remains in its open state (the summary window is already opened)** when we visit the website again either from the daily.dev extension feed or by reloading.

Here's a screen recording:

https://user-images.githubusercontent.com/11731837/173064428-e81078f2-4c22-4201-8489-75316f3565e1.mp4

2. When opening the comment dialog, I **can't see any cursor blinking indicator** to let me know that we can actually type something there.

Here's a screen recording:

https://user-images.githubusercontent.com/11731837/173065518-b28c606e-3c28-4560-8559-ff2ca2ec2d9c.mp4

### Expected Behavior

1. The summary window should be closed initially no matter how we used it last (be it open or closed state).
2. The comment dialog should have a blinking cursor for both accessibility and UX concerns. This makes it difficult to select, cut, replace comment text while writing it.

### Steps to Reproduce Issue

For "Open summary issue":

1. Open an article from the feed.
3. Click on "Open summary" (left chevron icon) to expand the widget.
4. Close the article tab.
5. Open the same article again from the feed and you will see the widget is already expanded.

For comment issue:

1. Click on "Start the discussion".
2. Try commenting or clicking on the modal that appears.

### Solution

1. The "Open summary" state should be set to `false` if applicable. The **widget should be in unexpanded view by default**.
2. The comment modal's input field **should automatically focus and show the blinking cursor**.

#### Operating System
macOS 12.4

#### Version of Daily
4.16.3

#### Browser
Firefox Developer Edition (102.0b6)

**Comment by idoshamun** [comment]
Hi @Kvaibhav01,

Thanks for the suggestion! :)

1. The companion remembers its last state and restores it when on a new window. Hence, you see it open every time.
2. We will fix the comment modal cursor! :)

**Comment by sshanzel** [comment]
Hi @Kvaibhav01,

Thank you for amazingly writing your concerns. For item number 2, it should be fixed with our latest release (v3.16.9). The issue was, that we used to use a div contenteditable due to its native capability of dynamically changing height. But after further investigation, there's been an open bug in FF open for more than 4 years, so we decided to use a textarea instead to provide a better user experience with our Companion.
